src/components/EmojiMenu.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { gemoji as gemojies } from "gemoji";
+import FuzzySearch from "fuzzy-search";
+
+import BlockMenu, { Props } from "./CommonBlockMenu";
+import EmojiMenuItem from "./EmojiMenuItem";
+
+const searcher = new FuzzySearch(gemojies, ["names"], {
+  caseSensitive: true,
+  sort: true,
+});
+
+type Emoji = {
+  name: string;
+  title: string;
+  emoji: string;
+  attrs: { markup: string; "data-name": string };
+};
+
+class EmojiMenu extends React.Component<
+  Omit<
+    Props<Emoji>,
+    | "renderMenuItem"
+    | "items"
+    | "onLinkToolbarOpen"
+    | "embeds"
+    | "onClearSearch"
+  >
+> {
+  get items(): Emoji[] {
+    const { search = "" } = this.props;
+
+    const n = search.toLowerCase();
+    const result = searcher.search(n).map(item => {
+      const name = item.names[0];
+      return {
+        ...item,
+        name: "emoji",
+        title: name,
```

Comment:
Is it possible to use "description" from gemoji here so that the text doesn't contain "_"?

Reply:
maybe not.. gemoji description longer than name

```json
{
  "emoji": "😍",
  "names": ["heart_eyes"],
  "tags": ["love", "crush"],
  "description": "smiling face with heart-eyes",
  "category": "Smileys & Emotion"
}
```

Reply:
Mm, maybe we can humanize the name then? (Capitalize, remove underscores)

Feels very strange to show underscores in user facing UI. I know GitHub does it, but GitHub is weird.

Reply:
maybe you are right, i will try it later.

Reply:
![image](https://user-images.githubusercontent.com/6839576/132095290-170aeda3-2583-4fca-a42d-2ff7de7ff814.png)